vcbuild.bat

```diff
@@ -671,11 +671,6 @@ echo running lint-js
 %node_exe% tools\node_modules\eslint\bin\eslint.js --cache --report-unused-disable-directives --rule "linebreak-style: 0" --ext=.js,.mjs,.md .eslintrc.js benchmark doc lib test tools
 goto lint-md-build
 
-:lint-js-ci
```

Comment:
There are other references to this label in this file that should be tidied up.

Reply:
Ooh, I thought I checked that, but obviously not! Fixed in a fixup commit....